sorc/ncep_post.fd/CALWXT.f

```diff
@@ -136,16 +142,20 @@ SUBROUTINE CALWXT_POST(T,Q,PMID,PINT,HTM,LMH,PREC,ZINT,IWX,ZWET)
 !
       IF (TCOLD(I,J).EQ.T(I,J,LMHK).AND.TDCHK.LT.6.0) THEN
         TDCHK=TDCHK+2.0
-        GOTO 760
+!       GOTO 760
+        cycle looptcold
       ENDIF
+      exit looptcold
+      enddo looptcold
   800 CONTINUE
```

Comment:
@BoCui-NOAA  Would it work with:
-------------------------
do
  ...
if ( ...)
   cycle
else
   exit
endif
enddo